examples/README.md

```diff
@@ -0,0 +1,191 @@
+# Examples
+
+This permanent W3ID is meant to be a place to host examples for publishing things on w3id.org.
+
+* [.htaccess](#htaccess)
+  * [What is .htaccess?](#what-is-htaccess)
+  * [Put ID info and maintainer info inside .htaccess](#put-id-info-and-maintainer-info-inside-htaccess)
+  * [Quick intro to URL writing rules](#quick-intro-to-url-rewriting-rules)
+  * [Example 1: Minimalist (grouping)](#example-1-minimalist-grouping)
+  * [Example 2: Supporting multiple media types (MIME types)](#example-2-supporting-multiple-media-types-mime-types)
+  * [Example 3: Dealing with query string](#example-3-dealing-with-query-string)
+* [README.md](#readmemd)
+* [Publish vocabularies with W3ID](#publish-vocabularies-with-w3id)
+
+## .htaccess
+
+`.htaccess` file is the key for the working of URL redirection service of W3ID. Without it, redirection cannot be done.
+
+### What is `.htaccess?`
+
+From [Wikipedia](https://en.wikipedia.org/wiki/.htaccess):
+> An .htaccess (hypertext access) file is a directory-level configuration file
+supported by several web servers, used for configuration of website-access
+issues, such as URL redirection, URL shortening, access control (for different
+web pages and files), and more. The 'dot' (period or full stop) before the
+file name makes it a hidden file in Unix-based environments. 
+
+In W3ID context, it is used primarily for URL redirection. The `.htaccess` file is where you can put URL rewriting rules in. A set of URL rewriting rules will work together and effectively made URL
+redirection happen.
+
+### Put ID info and maintainer info inside .htaccess
+
+You are encouraged to put a breif ID info and a maintainer info in the comment
+(lines staring with `#` character) of a `.htaccess` file.
+
+```ApacheConf
+# Example
+#
+# https://w3id.org/example redirects to https://example.com/
```

Comment:
This is not correct.

Reply:
Thank you. I have fixed all the instances that mentioned https://w3id.org/example wrongly.

In that example, I replace it with w3id.org/examples/simple instead.

For .htaccess in examples/, I am not sure where to redirect it to.